src/utils/xhr-utils.js

```diff
@@ -408,3 +408,27 @@ export function assertSuccess(response) {
     throw err;
   });
 }
+
+/**
+ * Returns a promise resolving to a string identity token if the element
+ * contains the 'crossorigin' attribute and the amp-viewer-assistance extension
+ * is present. Resolves to undefined otherwise.
+ * @param {!Window} win
+ * @param {!Element} element
+ * @return {!Promise<string|undefined>}
+ */
+export function getViewerAuthTokenIfAvailable(win, element) {
```

Comment:
The `win` argument is no longer needed, but let's address in a follow-up.

Reply:
Sounds good!